weather10kb.js

```diff
@@ -25,18 +25,21 @@ function Weather10kbRequest(request) {
           apiKey: process.env.GOOGLE_API_KEY,
           formatter: null         // 'gpx', 'string', ...
         });
+        var countryCode = 'US';
```

Comment:
This is already the case, but not until later in the code (if the geocoding request returns a valid match). US simply serves as a fallback (and to ensure the variable is initialized properly).

I don’t see how I can reliably determine the user’s location before running through the geocoder.   If you want me to make enhance that further, I’m sure that can be done as well. :)